capacitor.config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "appId": "co.craftacademy.gnosis",
```

Comment:
I would give it another identifier...

Reply:
Fixed according to convention in combination with current desktop deployment site.